build/Linux+BSD/portable/RecipeArm

```diff
@@ -0,0 +1,86 @@
+#!/bin/bash
+
+# This file is designed to run in debian jessie docker images
+
+main() {
+
+  set -e # Halt on error
+  set -x # Be verbose
+
+  case $1 in
+    "--fetch-dependencies-only")
+      fetchDependencies "$@"
+      ;;
+    "--build-only")
+      shift #need to adjust parameter list so $makefile_overrides start at $1
+      build "$@"
+      ;;
+    "--package-only")
+      package "$@"
+      ;;
+    *)
```

Comment:
I figured out how to avoid the wget, actually.  I can use the docker building command ADD which can use url as input.  So I'll just grab the file RecipeArm using ADD, and avoid installing wget :)